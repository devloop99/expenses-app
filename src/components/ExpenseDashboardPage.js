import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseFilterInput from "./ExpenseFilterInput";

const ExpenseDashboardPage = () => (
  <div>
    <ExpenseFilterInput />
    <ExpenseList />
  </div>
);

export default ExpenseDashboardPage;
