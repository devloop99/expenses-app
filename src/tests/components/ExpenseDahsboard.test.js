import ExpenseDashboardPage from "../../components/ExpenseDashboardPage";
import { shallow } from "enzyme";
import React from "react";

test("should dashboard render ", () => {
  const wrapper = shallow(<ExpenseDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
