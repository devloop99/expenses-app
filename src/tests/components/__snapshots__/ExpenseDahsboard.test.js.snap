// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should dashboard render  1`] = `
<div>
  <Connect(ExpenseFilterInput) />
  <Connect(ExpenseList) />
</div>
`;
